python/tvm/script/builder/tir/op.py

```diff
@@ -53,7 +53,7 @@
 from tvm.tir.op import atan2, clz, comm_reducer, infinity, reinterpret
 from tvm.tir.op import min_value, max_value, if_then_else
 from tvm.tir.op import call_packed, call_extern
-from tvm.tir.expr import Select, Ramp, Broadcast, Shuffle
+from tvm.tir.expr import Select, Ramp, Broadcast, Shuffle, StringImm
```

Comment:
There is no particular actionable item to do on this line, but I just wanted to share some context on StringImm: `StringImm` is only used when we want to pretend that a string is a PrimExpr (which usually represents numbers)

It's usually unavoidable in every single case, but if we can, let's try our best to let users write as fewer as `T.StringImm` as possible.

Reply:
Sure. I added it just for usage in `test_builder_basic.py`. In fact, those raw `str` can be converted to `StringImm` in parser as before. We can withdraw this addition here.